python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py

```diff
@@ -597,6 +603,82 @@ def quantize(model: Model,
+                metadata = {
+                    'ModelType': 'KerasModel',
+                    'checkpoint': 'saved_weight.ckpt'
+                }
+                yaml.safe_dump(metadata, f)
+            checkpoint_path = path / metadata['checkpoint']
+            model.save_weights(checkpoint_path)
+
+    @staticmethod
+    def load(path, model: Optional[Model] = None, device=None):
+        """
+        Load a model from local.
+
+        :param path: Path to model to be loaded. Path should be a directory.
+        :param model: Required FP32 model to load tensorflow model, it is needed if you
+               accelerated the model with accelerator=None by InferenceOptimizer.trace/
+               InferenceOptimizer.quantize. model should be set to None if you choose
+               accelerator="onnxruntime"/"openvino".
+        :param device: A string represents the device of the inference. Default to None.
+               Only valid for openvino model, otherwise will be ignored.
+        :return: Model with different acceleration(None/OpenVINO/ONNX Runtime) or
+                 precision(FP32/FP16/BF16/INT8).
+        """
+        import yaml
+        path = Path(path)
+        if not path.exists():
+            invalidInputError(False, "{} doesn't exist.".format(path))
+        meta_path = path / "nano_model_meta.yml"
+        if not meta_path.exists():
+            invalidInputError(False, "File {} is required to load model.".format(str(meta_path)))
+        with open(meta_path, 'r') as f:
+            metadata = yaml.safe_load(f)
+        model_type = metadata.get('ModelType', None)
+        if model_type == 'KerasOpenVINOModel':
+            invalidInputError(model is None,
+                              "Argument 'model' must be None for OpenVINO loading.")
+            return load_openvino_model(path, framework='tensorflow', device=device)
+        if model_type == 'KerasONNXRuntimeModel':
+            invalidInputError(model is None,
+                              "Argument 'model' must be None for ONNX Runtime loading.")
+            return load_onnxruntime_model(path, framework='tensorflow')
+        if model_type == 'KerasQuantizedModel':
+            return load_inc_model(path, model, framework='tensorflow')
+        if isinstance(model, Model):
+            # typically for keras Model
+            model = copy.deepcopy(model)
+            checkpoint_path = metadata.get('checkpoint', None)
+            if checkpoint_path:
+                checkpoint_path = path / metadata['checkpoint']
+                model.load_weights(checkpoint_path)
+                return model
+            else:
+                invalidInputError(False, "Key 'checkpoint' must be specified.")
+        else:
+            invalidInputError(False,
+                              "ModelType {} or argument 'model={}' is not acceptable for tensorflow"
+                              " loading.".format(model_type, type(model)))
+
```

Comment:
We may need to wrap model here again because we only saved the unwrapped model, to do this, we should always require user pass original model

Reply:
Has changed to always require user pass  original model to wrap model atribute.